rlbot_gui/story/story_runner.py

```diff
@@ -8,6 +8,7 @@
 import eel
 from PyQt5.QtCore import QSettings
 
+from persistence.settings import load_launcher_settings, launcher_preferences_from_map
```

Comment:
I get an import error here, probably should be
```suggestion
from rlbot_gui.persistence.settings import load_launcher_settings, launcher_preferences_from_map
```

Reply:
Thanks, good catch!